src/js/app.js

```diff
@@ -6,6 +6,8 @@ let connectedBlueThemeOption = document.getElementById('connectedBlue');
 let clearThemeOption = document.getElementById('clear');
 let connectedDarkBlueThemeOption = document.getElementById('connectedDarkBlue');
 
+const QUOTES_API_URL = "https://api.quotable.io/random";
```

Comment:
**Change requested**

I just looked at the API. Maybe we add the tags param and update the URL to `https://api.quotable.io/random?tags="Wisdom|Happiness|Inspirational"`

I noticed that not all quotes were "motivational" so that'll ensure we still get quotes that match the previously displayed quotes.